docs/server/production-deploy.md

```diff
@@ -98,6 +123,40 @@ The main technical hurdle is that each task can have its own `ScheduleToStart` t
 This is why we recommend tracking `ScheduleToStart` latency for determining if the task queue has a backlog (aka Workers are under-provisioned for a given Task Queue).
 We do plan to add features that give more visibility into the task queue state in future.
 
+### FAQ: High Availability cluster configuration
+
+You can set up a high availability deployment by running more than one instance of the server. Temporal also handles [membership and routing](https://docs.temporal.io/blog/workflow-engine-principles/#membership-and-routing-1350). You can find more details in [the `clusterMetadata` section of the Server Configuration reference](https://docs.temporal.io/docs/server/configuration/#clustermetadata).
+
+```yaml
+clusterMetadata:
+  enableGlobalNamespace: false
+  failoverVersionIncrement: 10
+  masterClusterName: "active"
+  currentClusterName: "active"
+  clusterInformation:
+    active:
+      enabled: true
+      initialFailoverVersion: 0
+      rpcAddress: "127.0.0.1:7233"
+```
+
+### FAQ: Multiple deployments on a single cluster
+
+You may sometimes want to have multiple parallel deployments on the same cluster, eg:
+
+- when you want to split Temporal deployments based on namespaces, e.g. staging/dev/uat, or for different teams who need to share common infrastructure.
+- when you need a new deployment to change `numHistoryShards`.
+
+**We recommend not doing this if you can avoid it**. If you need to do it anyway, double-check the following:
+
+- Have a separate persistence (database) for each deployment
+- Cluster membership ports should be different for each deployment (they can be set through environment variables). For example:
+  - Temporal1 services can have 7233 for frontend, 7234 for history, 7235 for matching
+  - Temporal2 services can have 8233 for frontend, 8234 for history, 8235 for matching
+- There is no need to change gRPC ports.
+
+[More details about the reason here](https://github.com/temporalio/temporal/issues/1234).
```

Comment:
@meetchandan last call for anything else you wanna add before we merge this update! ty for feedback so far